web3_proxy/src/frontend/mod.rs

```diff
@@ -58,6 +63,14 @@ pub async fn serve(
         .time_to_live(Duration::from_secs(1))
         .build();
 
+    // Move this to the mod.rs (frontend) server, this does not have to live inside the app
+    // These responses are pretty large, so we will only save up to 1000 at a time. They can live for longer, however
+    let influx_cache: InfluxResponseCache = CacheBuilder::new(1_000_000)
```

Comment:
@WyseNynja capacity is very high, but having it less made the cache not work out